tools/update-copyright-notices.sh

```diff
@@ -10,4 +10,4 @@ for file in $(find android ios linux mac src windows -regex '.*\.\(cpp\|h\|mm\)'
 		sed -re 's/(\*.*Copyright.*[^-][0-9]{4})(\s*-\s*\b[0-9]{4})?\s*$/\1-'"${YEAR}"'/' -i "${file}"
```

Comment:
Here and line 6 are also brought into question, perhaps?

Reply:
Do you mean the regexp? I've adjusted the other instance because it's a different format (multi-line value in debian/control)

Reply:
I just meant does the same reasoning - splitting the Copyright line - also apply to the file headers on the source, as well as the debian copyright message.  I'm not sure most of them credit the contributors (although I've tended to add myself).

Reply:
Ah, yeah.
I've also thought about those multiple times, but didn't come to a good solution.

Yes, most contributors are missing.

- We could add everyone based on git history, but this will get out of hands. `src/utils.cpp` would show 33 author lines, from a quick glance (`git log  --format='format:%an' src/util.cpp  | sort -u | wc -`).
- Listing only Volker tends to become less valid over time.
- Dropping authors altogether and listing them somewhere else instead (`src/utils.cpp` does) would also imply to drop Volker's name, which does not sound nice to me.
- Maybe a compromise would be `Volker Fischer, The Jamulus Development Team and contributors`?

Reply:
Probably worth raising as a separate issue for now.